guide/multiline.md

```diff
@@ -0,0 +1,92 @@
+# Multiline Text
+
+Text can span multiple lines as long as it is indented by at least one space.
+Line breaks inside of multiline text are preserved in the final text value.
+
+```
+single = Text can be written in a single line.
+
+multi = Text can also span multiple lines
+    as long as each new line is indented
+    by at least one space.
+
+block =
+    Sometimes it's more readable to format
+    multiline text as a "block", which means
+    starting it on a new line. All lines must
+    be indented by at least one space.
+```
+
+In almost all cases, patterns start at the first non-blank character and
+end at the last non-blank character. In other words, the leading and
+trailing blanks are ignored. There's one exception to this rule, due to
+another rule; see the `multiline2` message below.
+
+```
+leading-spaces =     This message's value starts with the word "This".
+leading-lines =
+
+
+    This message's value starts with the word "This".
+    The blank lines under the identifier are ignored.
+```
+
+In multiline patterns, the maximum indent common to all indented lines is
```

Comment:
OK, I see what you mean. My brain expects *minimum*, because that's the shortest (minimum) among all the indentations. 

I also don't see that as 0, though; why do you? Are you thinking of this?

```
multi = Text can also span multiple lines
    as long as each new line is indented
    by at least one space.
```

For example, I don't expect it to be zero here:

```
block =
    Sometimes it's more readable to format
    multiline text as a "block", which means
    starting it on a new line. All lines must
    be indented by at least one space.
```

Going back to the problem, I still wonder if "minimum indent" would be clearer?